web/pdf_find_controller.js

```diff
@@ -1133,6 +1133,7 @@ class PDFFindController {
       source: this,
       state,
       previous,
+      entireWord: this.#state?.entireWord ?? null,
```

Comment:
Is `null` actually an appropriate fallback value, or would `false` make more sense?
```suggestion
      entireWord: this.#state?.entireWord ?? false,
```

Reply:
I initially prefer to leave it `undenfined` since the controller should take responsibility to prepare it, then I choose `null` to follow local convention (I guess) like how `rawQuery` is handled.

What do you think if just remove the `?? null` fallback?

Reply:
I suppose that we can leave this as-is then, however please address the other feedback such that we can review this.